### ISSUE TITLE
MissingOutputException

### ISSUE THREAD

**Issue by pvtodorov** [perslab/CELLECT#75]
- Attempting to run CELLECT results in a `MissingOutputException`
- setting filesystem latency to a higher number, (e.g. `--latency-wait 60`) as suggested in the error message, does not resolve the issue 
- no other errors or leads provided

```
Waiting at most 60 seconds for missing files.
MissingOutputException in line 180 of /Users/nmq407/Code/CELLECT/cellect-ldsc.snakefile:
Job Missing files after 60 seconds:
/Users/nmq407/cellect_inputs/CELLECT-LDSC/results/prioritization.csv
This might be due to filesystem latency. If that is the case, consider to increase the wait time with --latency-wait.
Job id: 1 completed successfully, but some output files are missing. 1
```

A zipped directory containing sumstat input, esmu input, config file, terminal traces, and CELLECT output is attached
[cellect_inputs.zip](https://github.com/perslab/CELLECT/files/7511915/cellect_inputs.zip)

**Comment by pvtodorov** [comment]
Resolved: only have alphanumerics and single underscores in the specificity ids.
Should be incorporated and caught as a check in code.

**Comment by pvtodorov** [comment]
should be solved by #76
